firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- [fixed] Fixed calculation of SQLite database size on Android 9 (P) devices.
```

Comment:
"Android 9 (P)" is a nonstandard way of referring to this. Android documentation calls this "Android 9 Pie". We should follow since the changelog gets rolled up into our public release notes.

Reply:
Done.